main.go

```diff
@@ -12,7 +12,7 @@ import (
 
 var (
 	// Version is the main version number that is being run at the moment.
-	Version = "0.0.2"
+	Version = "0.0.3"
```

Comment:
This one is still two, we update once we release the 0.0.2
```suggestion
	Version = "0.0.2"
```